### PR TITLE
doc: bluetooth: Update overview

### PR DESCRIPTION
**Pull request by carlescufi**
Update the Bluetooth overview with additional content.

Signed-off-by: Carles Cufi <carles.cufi@nordicsemi.no>

### CODE REVIEW

File: doc/connectivity/bluetooth/overview.rst [hunk 1]
```diff
@@ -93,3 +101,24 @@ Bluetooth stack.
     * Local controller support as a virtual HCI driver
 
   * Verified with multiple popular controllers
+
+* LE Audio (Auracast) in Host and Controller
```
Comment (Thalley) [inline doc/connectivity/bluetooth/overview.rst:105]: LE Audio and Auracast cannot be used interchangeably. 

Auracast is only the broadcast part (and the full name is Auracast broadcast audio: https://www.bluetooth.com/auracast/)

File: doc/connectivity/bluetooth/overview.rst [hunk 2]
```diff
@@ -93,3 +101,24 @@ Bluetooth stack.
...
+
+      * Unicast server and client
+      * Broadcast source and sink
+      * Broadcast assistant
+      * Scan delegator
+
+    * Common Audio Profile
+
+      * Acceptor
+      * Initiator and Commander
```
Comment (Thalley) [inline doc/connectivity/bluetooth/overview.rst:120]: The initiator is in PR, and Commander is not yet started

File: doc/connectivity/bluetooth/overview.rst [hunk 3]
```diff
@@ -93,3 +101,24 @@ Bluetooth stack.
...
+
+  * Volume Control Profile, Microphone Control Profile
+  * Call Control Profile, Media Control Profile
+  * Coordinated Set Identification Profile
```
Comment (Thalley) [inline doc/connectivity/bluetooth/overview.rst:124]: These are also part of the Generic Audio Framework (should be indented)

File: doc/connectivity/bluetooth/overview.rst [hunk 4]
```diff
@@ -58,11 +59,17 @@ Bluetooth stack.
 
     * Peripheral & Central
     * Observer & Broadcaster
+    * Multiple PHY support (2Mbit/s, Coded)
+    * Extended Advertising
+    * Periodic Advertising (including Sync Transfer)
```
Comment (Thalley) [inline doc/connectivity/bluetooth/overview.rst:64]: The Zephyr controller does not support sync transfer yet

**Review by Thalley — CHANGES_REQUESTED** [review]

**Comment by jori-nordic** [comment]
if we are updating the docs as part of this release, can we also merge this? https://github.com/zephyrproject-rtos/zephyr/pull/54124

File: doc/connectivity/bluetooth/overview.rst (hunk 1 above)
Comment (carlescufi) [reply doc/connectivity/bluetooth/overview.rst:105]: Thanks, will remove

File: doc/connectivity/bluetooth/overview.rst (hunk 2 above)
Comment (carlescufi) [reply doc/connectivity/bluetooth/overview.rst:120]: Thanks

File: doc/connectivity/bluetooth/overview.rst (hunk 3 above)
Comment (carlescufi) [reply doc/connectivity/bluetooth/overview.rst:124]: Oh right! will fix

File: doc/connectivity/bluetooth/overview.rst (hunk 4 above)
Comment (carlescufi) [reply doc/connectivity/bluetooth/overview.rst:64]: Very true, oversight on my part.

**Comment by carlescufi** [comment]
> if we are updating the docs as part of this release, can we also merge this? #54124

I approved it and requested a review from @jhedberg

File: doc/connectivity/bluetooth/overview.rst (hunk 4 above)
Comment (carlescufi) [reply doc/connectivity/bluetooth/overview.rst:64]: That's in the Host section

**Review by jhedberg — DISMISSED** [review]

File: doc/connectivity/bluetooth/overview.rst [hunk 5]
```diff
@@ -93,3 +101,23 @@ Bluetooth stack.
     * Local controller support as a virtual HCI driver
 
   * Verified with multiple popular controllers
+
+* LE Audio in Host and Controller
+
+  * Full support for Isochronous channels
```
Comment (Thalley) [inline doc/connectivity/bluetooth/overview.rst:107]: Controller only has limited support for Isochronous channels so far - Host can be considered feature-complete, and only lacks from test commands implemented in https://github.com/zephyrproject-rtos/zephyr/pull/53945

**Comment by hermabe** [comment]
Should the ICS tables be updated as well? I dont think they have been updated in a while?

**Comment by carlescufi** [comment]
> Should the ICS tables be updated as well? I dont think they have been updated in a while?

Yes, but for that I need more time and the help of @cvinayak and @Thalley. Let's do that in a separate PR.

**Review by Thalley — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**